creme/creme_core/static/creme_core/js/widgets/dialog/popover.js

```diff
@@ -72,7 +72,7 @@ creme.dialog.Popover = creme.component.Component.sub({
         var dialog = this._dialog;
         var close = this.close.bind(this);
 
-        dialog.css('visiblility', 'hidden');  // hide dialog before positioning
+        dialog.css('visibility', 'hidden');  // hide dialog before positioning
```

Comment:
In a separated commit which I'll backport.

Reply:
Done here https://github.com/HybirdCorp/creme_crm/pull/919